tools.js

```diff
@@ -59,6 +59,9 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	var init = typeof global.Symbol === 'function' ? Symbol('init') : '_init';
```

Comment:
Symbols are just another namespace, but for some reason they're implemented in a horrible unwieldy way. Why not just use a prefix?